kite-tools-parent/kite-tools/src/main/java/org/kitesdk/cli/Main.java

```diff
@@ -96,9 +98,11 @@
     jc.addCommand("schema", new SchemaCommand(console));
     jc.addCommand("info", new InfoCommand(console));
     jc.addCommand("show", new ShowRecordsCommand(console));
+    jc.addCommand("obj-schema", new ObjectSchemaCommand(console));
     jc.addCommand("csv-schema", new CSVSchemaCommand(console));
     jc.addCommand("csv-import", new CSVImportCommand(console));
-    jc.addCommand("obj-schema", new ObjectSchemaCommand(console));
+    jc.addCommand("json-schema", new JSONSchemaCommand(console));
+    jc.addCommand("json-import", new JSONImportCommand(console));
```

Comment:
I was talking to @busbey and it'd be nice if we could have a single `import` command and you could pass it an option of JSON, CSV, or an InputFormat. Do you think that would be doable?